pkg/scheduler/plugins/rescheduling/node_utilization_util.go

```diff
@@ -139,20 +140,27 @@ func getScoreForNode(index int, nodeUtilizationList []*NodeUtilization) float64
 // sortPods return the pods in order according the priority and QoS
 func sortPods(pods []*v1.Pod) {
 	cmp := func(i, j int) bool {
+		// sort by pod priority
 		if pods[i].Spec.Priority == nil && pods[j].Spec.Priority != nil {
 			return true
 		}
 		if pods[j].Spec.Priority == nil && pods[i].Spec.Priority != nil {
 			return false
 		}
+		// sort by pod Qos
 		if (pods[j].Spec.Priority == nil && pods[i].Spec.Priority == nil) || (*pods[i].Spec.Priority == *pods[j].Spec.Priority) {
-			if v1qos.GetPodQOS(pods[i]) == v1.PodQOSBestEffort {
-				return true
+			if v1qos.GetPodQOS(pods[i]) != v1qos.GetPodQOS(pods[j]) {
```

Comment:
Please simplify this piece of code for the following reasons:
1. It's hard to get the points for too many levels of if judgements.
2. if there is just a return with true or false, I think it is not necessary to write like the above, just return the results of the judgements.